.gitignore

```diff
@@ -7,6 +7,7 @@
 *.userprefs
 *.sln.docstates
 .vs/
+.dotnet/
```

Comment:
This change is unexpected.

Reply:
It is intentional. It belongs upstream, but isn't present.

---
In reply to: [233158234](https://github.com/dotnet/roslyn/pull/31129#discussion_r233158234) [](ancestors = 233158234)

Reply:
> It is intentional. It belongs upstream, but isn't present.

I see it in dev16.0-preview2. Consider reverting the change in the feature branch to avoid unnecessary merge conflicts.

---
In reply to: [233255602](https://github.com/dotnet/roslyn/pull/31129#discussion_r233255602) [](ancestors = 233255602,233158234)